Tasks/UseRubyVersionV0/Tests/L0FindVersionInLinuxCache.ts

```diff
@@ -0,0 +1,41 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+import fs = require('fs');
+import os = require('os');
+import tool = require('vsts-task-tool-lib/tool');
+
+let taskPath = path.join(__dirname, '..', 'main.js');
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+tr.setInput('versionSpec', '2.5');
+tr.setInput('addToPath', 'false');
+
+process.env['AGENT_VERSION'] = '2.116.0';
+
+// provide answers for task mock
+let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
```

Comment:
nit: can we delete this if it isn't needed, or at least remove the type assertion (`<ma.TaskLibAnswers>`)?